.github/workflows/tests.yml

```diff
@@ -81,13 +77,13 @@ jobs:
       # Runs a set of commands using the runners shell; THIS DOESN'T WORK
       - name: deploy
         run: |
-          zip -j deploy.zip ./src/module1 #--> Zip the Code As we know lambda function accept the zip file.
-          aws lambda update-function-code --function-name=test --zip-file=fileb://deploy.zip 
+          zip -j deploy.zip ./src/module1/* #--> Zip the Code As we know lambda function accept the zip file.
+          aws lambda update-function-code --function-name=lambda-gh-action-test --zip-file=fileb://deploy.zip 
```

Comment:
How should we scale this so we can deploy multiple lambda functions?

Maybe make a json file and have a script run here to handle deployment of each individual function